submissions/Articles/global-payouts-non-profits.md

```diff
@@ -0,0 +1,28 @@
+
+## Overcome Geographical Barriers with Global Payout Solutions
+
+When it comes to funding money overseas, a common issue that nonprofits and community initiatives face is that are restricted to only sending money to certain countries. For example, [Paypal only supports 25 curriences and 200 countries](https://www.paypal.com/sm/webapps/mpp/country-worldwide). This type of restriction can make it difficult for nonprofits, especially those whose mission is to aid people in need, to provide financial funding. Unlike traditional banks, global payout platforms like [Chimoney allows users make transactions anywhere in the worldwide](https://chimoney.io/payouts/). This can put nonprofits as ease as they would be able to send money overseas and in turn help, their intended audience.   Case in point, [Dream Design community's use of Chimoney's payout feature has made it easier for them to send stipends to aspiring tech leaders worldwide](https://chimoney.io/blogs/chimoney-dreamax-partnership-empowering-tech-enthusiasts/). Now no geographical barriers aren’t the only way global payouts can empower your community initiative or nonprofit. Let’s check out another one.
+
+## How Global Payouts Can Make Your Transactions Quick
+
+As cliche as it is to say, “Time is money”, it truly is important for users to get their money on time. Sadly, traditional banking institutions tend to have long delivery processes. For example, [Wise users receives their payment depending on the country and currency](https://wise.com/help/articles/2978029/how-long-does-it-take-to-receive-money-into-a-balance). This can put nonprofits at a disadvantage as it can cause the communities they serve to financially help them in a timely manner.  Unlike their traditional counterparts, global payout systems don’t have a waiting period. Once person sends money, the other person gets its instantly. Nonprofits can benefit greatly from this as they don’t have to worry about the communities they serve not getting the funding they sent.  For instance, [AfricaHacks' collaboration with Chimoney for their hackathon rewards to the winners due to having a feature where they can send people money in just one click](https://chimoney.io/blogs/5-types-of-people-who-should-use-chimoneys-bulk-gift-cards-feature/).  Now before you go, there’s just one more benefit of using global payout systems for nonprofit funding.
```

Comment:
- Global Payouts are offered by traditional banks, payment processors, fintech platforms so this part `global payout systems don’t have a waiting period` may be a bit confusing because global payout systems also include traditional banks. And a waiting period is most times always there but it depends with the payment provider one uses and where the user is paying out to, so depending on the provider the waiting time is shorter (that's Chimoney's selling point, the waiting times are always shorter compared to other providers and often times payment is instant) 

- AfriHacks is the hackathon from which the idea of Chimoney was built, after Chimoney was founded, consecutive AfriHacks hackathons were powered by Chimoney so for this sentence `For instance, [AfricaHacks' collaboration with Chimoney for their hackathon rewards to the winners due to having a feature where they can send people money in just one click](https://chimoney.io/blogs/5-types-of-people-who-should-use-chimoneys-bulk-gift-cards-feature/).` the accurate version may be

> [AfricaHacks uses Chimoney's bulk payout feature to reward their hackathon winners.With a single click, all winners receive their payments instantly and can redeem them through multiple options, including bank transfers, mobile money, airtime, and gift cards](https://chimoney.io/blogs/5-types-of-people-who-should-use-chimoneys-bulk-gift-cards-feature/).